presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -91,7 +91,8 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
 
             HiveClientConfig hiveClientConfig = new HiveClientConfig();
             HiveS3Config s3Config = new HiveS3Config();
-            HdfsConfiguration hdfsConfiguration = new HiveHdfsConfiguration(new HdfsConfigurationUpdater(hiveClientConfig, s3Config));
+            HiveWasbConfig wasbConfig = new HiveWasbConfig();
```

Comment:
No need to create a separate variable, just inline it where it is used like you do elsewhere.